library/src/main/java/com/mapzen/android/OverlayManager.java

```diff
@@ -35,11 +35,13 @@
   private static final String NAME_POLYGON = "mz_default_polygon";
   private static final String NAME_MARKER = "mz_default_point";
   private static final String NAME_START_PIN = "mz_route_start";
-  private static final String NAME_END_PIN = "mz_route_stop";
+  private static final String NAME_END_PIN = "mz_route_destination";
```

Comment:
This will be a backwards incompatible change, which is fine but needs to be coordinated with the stylesheet (scene file). Currently it's expecting `mz_route_stop` which was the old name. I think it is a good idea to change this in the app, too like you propose, so I'll push another change to the scene file for this.